packages/story-editor/src/components/canvas/frameElement.js

```diff
@@ -178,6 +199,16 @@ function FrameElement({ id }) {
   );
   const handleMediaClick = useDoubleClick(NOOP, handleMediaDoubleClick);
 
+  /**
+   * Announce keyboard options on element.
+   *
+   * Using a live region because an `aria-label` would remove
+   * any labels/content that would be read from children.
+   */
+  const handleFocus = useCallback(() => {
+    speak(FRAME_ELEMENT_MESSAGE);
```

Comment:
💭 The background element does not have a floating menu, so I think this needs to be prevented for the background element.